mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -1332,167 +1482,187 @@ class spawn_point_types {
-		name = "Dust-off and Medevac";
-		RESPAWN_SHORT;
+	// blackhorse /////////////////////////////////////////////////////////////////
+
+	class blackhorse_air_transport_all: parent_air_transport_all {
+		RESPAWN_MEDIUM;
+		lockTeams[] = LOCKED_BLACKHORSE;
+	};
+	class blackhorse_statics_105s {
+		name = "105 Artillery (Blackhorse)";
+		WRECK_LONG;
+		lockTeams[] = LOCKED_BLACKHORSE;
 		class categories {
-			class uh1 {
-				name = "UH-1 Medical";
-				icon = VEHICLE_ICON_HELO;
-				include[] = { { "uh1", "medical", "dustoff" } };
-				exclude[] = { { "pavn" } };
-			};
-			class ch47 {
-				name = "Ch-47 Medical";
-				icon = VEHICLE_ICON_HELO;
-				include[] = { { "ch47", "medical", "dustoff" } };
-				exclude[] = { { "pavn" } };
+			class m101 {
+				name = "M101";
+				icon = VEHICLE_ICON_STATIC;
+				include[] = { { "static", "artillery" } };
+				exclude[] = { {"mortar" } };
 			};
 		};
 	};
-
-	class air_transport_light {
-		name = "Light air";
+	class blackhorse_grnd_firesupport_custom {
+		name = "11th ACR (Blackhorse) Fire Support";
 		RESPAWN_SHORT;
-
+		lockTeams[] = LOCKED_BLACKHORSE;
 		class categories {
-			class cayuse {
-				name = "OH-6A";
-				icon = VEHICLE_ICON_HELO;
-				include[] = { {"oh6a", "transport" } };
-				exclude[] = { {"firesupport" } };
-			};/*
-			class hummingbird {
-				name = "Hummingbird";
-				icon = VEHICLE_ICON_HELO;
-				include[] = {"hummingbird"};
-			}*/
+			class tanks {
+				name = "Tanks";
+				icon = VEHICLE_ICON_ARMOUR;
+				include[] = { { "tank","army" } };
+				exclude[] = { { "arvn","usmc","pavn"} };
+			};
+			class m113 {
+				name = "M113";
+				icon = VEHICLE_ICON_ARMOUR;
+				include[] = { { "m113","firesupport" } };
+				exclude[] = { { "transport", "pavn" } };
+			};
+			class trucks {
+				name = "Trucks";
+				icon = VEHICLE_ICON_TRUCK;
+				include[] = { { "truck","firesupport" } };
+				exclude[] = { { "pavn" } };
+			};
 		};
 	};
 
-	class air_transport_medium {
-		name = "Medium Lift";
-		RESPAWN_SHORT;
+	// military police ////////////////////////////////////////////////////////////////
 
+	class mp_grnd_patrol {
+		name = "Patrol (Military Police)";
+		RESPAWN_MEDIUM;
+		lockTeams[] = LOCKED_MILITARY_POLICE;
 		class categories {
-			class uh1 {
-				name = "UH-1";
-				icon = VEHICLE_ICON_HELO;
-				include[] = { {"uh1","transport","army" } };
-				exclude[] = { {"firesupport", "7thcav","usmc","arvn"} };
+			class patrol {
+				name = "M151A1 Patrol";
+				icon = VEHICLE_ICON_CAR;
+				include[] = { { "m151a1", "armed","mp" } };
+				exclude[] = { { "transport","usmc","antitank","army" } };
+			};
+			class transport {
+				name = "M151A1 Transport";
+				icon = VEHICLE_ICON_CAR;
+				include[] = { { "m151a1", "transport","mp" } };
+				exclude[] = { { "armed","usmc","antitank","army" } };
 			};
 		};
 	};
+	class mp_air_transport_medium: parent_air_transport_medium
+	{
+		RESPAWN_SHORT;
+		lockTeams[] = LOCKED_MILITARY_POLICE;
+	};
 
-	class air_transport_heavy {
```

Comment:
now `unlocked_air_transport_heavy`, tempalte for it is `parent_air_transport_heavy`